product_variant_configurator/tests/test_product_variant_configurator.py

```diff
@@ -595,10 +595,13 @@ def test_product_template_write(self):
         self.assertTrue(self.product_template_no.product_variant_ids)
 
     def test_product_template_create(self):
+        # In the past, the product_template's name would
+        # be incorrectly set to the value of product_name
+        # if present in the context
         product = self.product_template.with_context(
             product_name="Context product name"
         ).create({"name": "Test"})
-        self.assertEqual(product.name, "Context product name")
+        self.assertEqual(product.name, "Test")
```

Comment:
This proves that removing it it's not correct, as the test have changed. You should keep it.

Reply:
I've restored the create method on product_template, in that case, and simply implemented the model_create_multi decorator as with the other creates.